synapse/lib/stormlib/cortex.py

```diff
@@ -20,18 +20,29 @@
         'storm': '''
         $apis = $lib.cortex.httpapi.list()
         if $apis {
-            $header = 'order iden                             owner                auth  runas  path'
-            $lib.print($header)
+            $config = (({
```

Comment:
Noodling this - we could put this type of configuration data into `cmdconf` and avoid having to parse it in storm.